.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+    - "3.4"
+install:
+    - "pip install pep8"
+script: "pep8 barcode/"
```

Comment:
One thing which might not be obvious: in addition to providing a ``.travis.yml`` file in the repo, you have to explicitly turn on Travis CI in the repo settings.

I noticed a couple of PEP-8 violations which Travis would have caught.  Not a huge deal, obviously, but probably worth turning it on.  Also if/when we eventually configure travis to run unit tests it'll be extra handy.